libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -181,7 +181,7 @@ const AP_Param::GroupInfo AP_SerialManager::var_info[] = {
 
 #if HAL_HAVE_SERIAL1
     // @Param: 1_PROTOCOL
-    // @DisplayName: Telem1 protocol selection
+    // @DisplayName: Serial1 protocol selection
```

Comment:
PIXHAWK and CUBE systems have TELEM1 and TELEM2 labeled on their cases.
I think it would be better to match the display accordingly.

Reply:
Many , many other autopilots have a different default protocol....

Reply:
To avoid discussion and thus speed this change into master I've removed the changes to the parameter "DisplayName" and just left in the core change to add support for the 12500 baud rate which is really the key change that users need.